spec/schema.json

```diff
@@ -320,6 +448,70 @@
         }
       }
     },
+    "GeneralPartnerData": {
+       "type": "object",
+       "properties": {
+        "country": {
+          "type": "string"
+        },
+        "date_effective_from": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "date_of_birth": {
+         "$ref": "#/definitions/DateOfBirth"
+        },
+        "etag": {
+          "type": "string"
+        },
+        "forename": {
+          "type": "string"
+        },
+        "former_names": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "governing_law": {
+          "type": "string"
+        },
+        "kind": {
+          "type": "string"
+        },
+        "legal_entity_register_name": {
+          "type": "string"
+        },
+        "legal_entity_registration_location": {
+          "type": "string"
+        },
+        "legal_form": {
+          "type": "string"
+        },
+        "nationality": {
```

Comment:
The screen suggests a second nationality is allowed. Can't see this catered for here.